### ISSUE TITLE
Cannot Configure the project.

### ISSUE THREAD

**Issue by Khawar-Iqbal** [breadwallet/brd-mobile#9]
`A problem occurred configuring project :WalletKitJava:corenative-android.`
> NDK not configured. Download it with SDK manager. Preferred NDK version is '21.1.6352462'. `

I've downloaded the ndk version of 21.1.6352462 but it is still not configuring it.

**Comment by DrewCarlson** [comment]
Can you verify if the `external/walletkit/WalletKitJava/local.properties` file was generated locally?  If not, copy it from the root directory into the `WalletKitJava` folder.

**Comment by Khawar-Iqbal** [comment]
will the project only run on macOS? it does not support windows? 
because im also getting the OS issue : 

`Caused by: java.lang.AssertionError: Unsupported OS: windows. Expression: false`

**Comment by DrewCarlson** [comment]
Correct, [WalletKit](https://github.com/blockset-corp/walletkit) only supports macOS hosts and this project relies on a locally built and linked copy.  Because WalletKit is not published to Maven Central, you'll have to build and host your own copy of the library to build from windows.
